plugins/flytekit-kf-pytorch/tests/test_elastic_task.py

```diff
@@ -95,3 +96,19 @@ def test_task(n: int):
         return n + 1
 
     test_task(n=1)
+
+
+@pytest.mark.parametrize("start_method", ["spawn", "fork"])
+def test_rdzv_configs(start_method: str) -> None:
+    """Test that rendezvous configs are passed to torch distributed."""
+    from torch.distributed.launcher.api import LaunchConfig
+
+    rdzv_configs = {"join_timeout": 10}
```

Comment:
Is it `join_timeout` or `timeout`, as specified [here](https://github.com/pytorch/pytorch/blob/18bcf62bbcf7ffd47e3bcf2596f72aa07a07d65f/torch/distributed/launcher/api.py#L91)? Or do both exist but they are something different?

Reply:
yeah, in addition to `timeout` there are other timeout params in [create_handler](https://github.com/pytorch/pytorch/blob/main/torch/distributed/elastic/rendezvous/dynamic_rendezvous.py#L1201)

Reply:
Would you be willing to document (or link to) the available config values in the `Elastic` dataclass' docstring? One has to look rather deep into the torch code base to find these and it would be nice to make this as simple as possible for our users.

Reply:
links added in the `Elastic` docstring https://github.com/flyteorg/flytekit/pull/1751/files#diff-b478185433e96420ea9cb3067d6ccfaaf881094c7c2c8e3b53b252a9128785adR127

Reply:
Thanks 🙏